tools/percolator/msgf2pin.xml

```diff
@@ -1,14 +1,29 @@
-<tool id="msgfplus2pin" name="MS-GF+ output to Pin converter" version="2.10">
+<tool id="msgfplus2pin" name="MS-GF+ output to Pin converter" version="3.0">
     <description>to create Percolator input files</description>
     <requirements>
-        <requirement type="package" version="2.10">percolator</requirement>
+        <requirement type="package" version="3.0">percolator</requirement>
     </requirements>
     <stdio>
         <exit_code range="1:"/>
     </stdio>
-    <command>python $__tool_directory__/msgf2pin.py
-             enzyme=$enzyme matches=$matches aa-freq=$aafreq PTM=$ptm PNGaseF=$pngase isotope=$isotope 
-             $pinout $target $decoy
+    <command>
+             msgf2pin -o $pinout -e $enzyme
+             #if $matches
+             -m $matches
+             #end if
+             #if $aafreq
+             -a
+             #end if
+             #if $ptm
+             -b
```

Comment:
@glormph there is a better way for booleans in Galaxy.
please add `truevalue="-b" falsevalue=""` to your param and here in the command just $ptm.

This will do the trick, no need to the if/else stuff.